src/unity/python/turicreate/util/__init__.py

```diff
@@ -717,19 +717,41 @@ def subprocess_exe(exe, args, setup=None, teardown=None,
 # Automatic GPU detection
 def _get_cuda_gpus():
     """
-    Returns a list of 0-based integer indices of available CUDA GPUs.
+    Returns a list of dictionaries, with the following keys:
+    - index (integer, device index of the GPU)
+    - name (str, GPU name)
+    - memory_free (float, free memory in MiB)
```

Comment:
`MiB` -> `MB`? Also same question about the next line.

Reply:
I don't feel strongly about this, so I'm OK switching it to MB. I think MiB is clearer since it's unambiguous, but again, I'm happy to change this.